fmc/tests/test_rtalias.rs

```diff
@@ -82,8 +82,8 @@ fn test_fht_info() {
 
     let data = hw.mailbox_execute(TEST_CMD_READ_FHT, &[]).unwrap().unwrap();
     let fht = FirmwareHandoffTable::read_from_prefix(data.as_bytes()).unwrap();
-    assert_eq!(fht.ldevid_tbs_size, 533);
-    assert_eq!(fht.fmcalias_tbs_size, 769);
+    assert_eq!(fht.ldevid_tbs_size, 544);
```

Comment:
didn't the size only increase by 9 rather than 11?

Reply:
Alignment added 2 bytes.

Reply:
got it